nvm.sh

```diff
@@ -13,7 +13,7 @@
 NVM_SCRIPT_SOURCE="$_"
 
 nvm_echo() {
-  command printf %s\\n "$*" 2>/dev/null
+  echo "$*" 2>/dev/null
```

Comment:
Why this change? This is problematic; the `command` is necessary to bypass any user aliases or functions, and the `printf` is more reliable than `echo` across shells.

Reply:
@ljharb 

I have described it in the description.

> I have changed command printf on echo because printf is slower than echo
(printf - 2ms, echo - 0.09ms)

What do you think about `command echo`?

Thanks

Reply:
We can certainly try it; I *think* i added automated tests around this one. Specifically the issue was about multiline input on specific shells, I think dash and sh.

Reply:
🤔  I have checked now and I was wrong.

`command printf` majorized by `command` so if we could not remove *command* I just revert this change

Thanks

Reply:
Sounds good; some slowdowns are inevitable for correctness, sadly, but we can still try to shave off time wherever we can :-)